test/socket.io.js

```diff
@@ -1546,9 +1546,7 @@ describe('socket.io', function(){
       var srv = http();
       var sio = io(srv);
       srv.listen(function() {
-        var addr = srv.listen().address();
-        var url = 'ws://localhost:' + addr.port + '?key1=1&key2=2';
-        var socket = ioc(url);
+        var socket = client(srv, {query: {key1: 1, key2: 2}});
```

Comment:
According this:
https://github.com/socketio/socket.io/blob/master/.github/PULL_REQUEST_TEMPLATE.md
[Improve code quality]